### ISSUE TITLE
The includes method didn't return valid value in Rails 5

### ISSUE THREAD

**Issue by elct9620** [thiagopradi/octopus#467]
Hi, all

I am working on a large project to upgrade Rails 4 to Rails 5. But we have some problem with `includes` method.

```ruby
User.using(:user01).first.user_sessions.includes(:user_wishes).find { |s| s }
```

In Rails 4, it will return `user_session` but in Rails 5 it returns `[]`  if we want to got correct value, we have to add `to_a` like below:

```ruby
User.using(:user01).first.user_sessions.includes(:user_wishes).to_a.find { |s| s }
```

This happens many times in our project, and we have to add `to_a` to temporary let it works.

**Comment by shyam-habarakada** [comment]
Does this happen only when you are using Octopus?

**Comment by elct9620** [comment]
Yes, but I think it is resolved after I try to add some patch for #478.
